tempesta_fw/connection.c

```diff
@@ -180,9 +182,8 @@ tfw_connection_send_srv(TfwSession *sess, TfwMsg *msg)
 	BUG_ON(srv_conn->sess && srv_conn->sess != sess);
 	srv_conn->sess = sess;
 
-	ss_send(sess->srv->sock, &msg->skb_list);
-
-	return 0;
+	/* XXX: will be refactored. */
+	return tfw_lb_send_msg(msg);
```

Comment:
Just noticed that tfw_connection_send_srv() returns int, but nobody (in http.c) is interested in it. It seems we should treat somehow problems on sending a message. Probably, we should initiate connection failovering...